test/ret_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule RetTest do
+  use Ret.DataCase
+  import Ecto.Query, only: [from: 2]
+  import Ret.TestHelpers
+  alias Ret.{Account}
+
+  describe "account deletion" do
+    test "deletes account, login, identity, oauthproviders, and api_credentials" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      test_account = create_account("test")
+
+      Account.set_identity!(test_account, "test identity")
+
+      %Ret.OAuthProvider{}
+      |> Ecto.Changeset.change(source: :discord, account: test_account, provider_account_id: "discord-test-user")
+      |> Ret.Repo.insert_or_update()
```

Comment:
This could probably simplified. Like so:
`Repo.insert(%OAuthProvider{source: :discord, account: test_account, provider_account_id: "discord-test-user"})`

Reply:
(A changeset isn’t required for an insert)